sample/HttpTrigger-CSharp-CustomRoute/function.json

```diff
@@ -5,7 +5,7 @@
             "name": "info",
             "direction": "in",
             "methods": [ "get" ],
-            "route": "csharp/products/{category:alpha?}/{id:int?}"
+            "route": "csharp/products/{category:alpha?}/{id:int?}/{extra?}"
```

Comment:
The key difference between this new route parameter and the previous one is that this one does not exist in the binding contract that is created from the POCO we're binding to. That is the scenario that did not work as the contract had no information about this binding argument.